src/controllers/UserController.ts

```diff
@@ -34,3 +35,32 @@ export default class UserController {
     })
   }
 }
+
+export const changeUserRole = async (req: Request, res: Response) => {
+  try {
+    const { userId } = req.params
+    const { newRole } = req.body
+
+    if (!userId || !newRole) {
+      return res.status(400).json({ message: 'Bad request' })
+    }
+
+    if (!['admin', 'buyer', 'seller'].includes(newRole)) {
```

Comment:
```suggestion
    if (![Role.ADMIN, Role.BUYER, Role.SELLER].includes(newRole)) {
```
We can use enum instead